SS.Integration.Adapter/Actors/Messages/StreamConnectedMsg.cs

```diff
@@ -12,12 +12,18 @@
 //See the License for the specific language governing permissions and
 //limitations under the License.
 
+using System;
+
 namespace SS.Integration.Adapter.Actors.Messages
 {
     internal class StreamConnectedMsg
     {
         public string FixtureId { get; set; }
 
         public string FixtureStatus { get; set; }
+
+        public DateTime TimeStamp { get; set; }
+
+        public bool IsStreamConnected { get; set; }
```

Comment:
I'm not sure why StreamConnectedMsg has IsStreamConnected flag??? 
The purpose of this message is to inform the recipient that the stream is successfully connected.